### PR TITLE
Public API update

### PR DESCRIPTION
**Pull request by andrew-boyarshin**
AndroidManifest.xml part of update. Have a look at main part of this update.

### CODE REVIEW

**Comment by zhuowei** [comment]
thanks - will take a look.

File: AndroidManifest.xml [hunk 1]
```diff
@@ -1,47 +1,59 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-      package="net.zhuoweizhang.mcpelauncher"
-      android:versionCode="43"
-      android:versionName="1.6">
+    package="net.zhuoweizhang.mcpelauncher"
```
Comment (zhuowei) [inline AndroidManifest.xml]: Why re-indent?

File: AndroidManifest.xml [hunk 2]
```diff
@@ -51,42 +63,78 @@
...
         </activity>
-        <activity android:name="net.zhuoweizhang.mcpelauncher.ManageAddonsActivity"/>
-        <activity android:name="net.zhuoweizhang.mcpelauncher.ManageScriptsActivity" android:label="@string/pref_zz_manage_scripts">
+        <activity android:name="net.zhuoweizhang.mcpelauncher.ImportTexturepackActivity" >
+            <intent-filter>
+                <action android:name="net.zhuoweizhang.mcpelauncher.action.SET_TEXTUREPACK" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+
+                <data android:mimeType="*/*" />
```
Comment (zhuowei) [inline AndroidManifest.xml]: Why not zip?

**Comment by andrew-boyarshin** [comment]
Re-indent - Eclipse.
Not zip - my careless mistake.

**Pull request closed without merging**